DATA_README.md

```diff
@@ -0,0 +1,50 @@
+# Cumulus - Data model
+
+Cumulus uses DynamoDB for database storage. While DynamoDB is a schemaless database storage system, Cumulus enforces schemas on record creation and update using the [`ajv` package](https://github.com/epoberezkin/ajv).
+
+The data models managed by Cumulus and their schema definitions can be found in the [Cumulus source code](./packages/api/models/schemas.js).
+
+## Documenting changes
+
+All changes to the database models should be noted in the [data model CHANGELOG](./DATA_CHANGELOG.md).
```

Comment:
```suggestion
All changes to the database models must be noted in the [data model CHANGELOG](./DATA_CHANGELOG.md). Tracking backwards compatibility will otherwise be extremely difficult.
```